tests/extensions-are-available/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '2'
+
+services:
+    postgres:
+        build:
+          context: ../../PostgresExtended/
+          dockerfile: Postgres-latest.Dockerfile
+        environment:
+          NODE_ID: 1 # Integer number of node
+          NODE_NAME: node1 # Node name
+          CLUSTER_NODE_NETWORK_NAME: postgres # (default: hostname of the node)
+          
+          POSTGRES_PASSWORD: monkey_pass
+          POSTGRES_USER: monkey_user
+          POSTGRES_DB: monkey_db
+          CLEAN_OVER_REWIND: 0
+          CONFIGS: "listen_addresses:'*',max_replication_slots:5"
```

Comment:
Redundant details about replication slots, better to remove

Reply:
ack